src/App.jsx

```diff
@@ -1,10 +1,55 @@
-import React from 'react';
-import './App.css';
+import { React, useState } from 'react';
+import axios from 'axios';
+
+const geocodingURI = 'http://api.openweathermap.org/geo/1.0/direct?q=';
+const weatherApiKey = process.env.REACT_APP_WEATHER_API_KEY;
 
 function App() {
+  const [geoData, setGeoData] = useState({});
+
+  const retrieveGeoData = async (city = '') => {
+    try {
+      const response = await axios.get(
+        `${geocodingURI}${city}&appid=${weatherApiKey}`
```

Comment:
Did you still want to hard-code for the U.S. for now? Right now the call returns coordinates for international locations as well.

Reply:
I left out USA b/c it seems to work for international locations and I think that would be okay!

Reply:
That makes sense, it's just that it makes it harder to adjust for duplicate city names when the search is worldwide. But if it works that's okay!

Reply:
This can definitely be refactored to include state and country as well later. Since we've abstracted this away, other components only need the lat and lon so as long as the refactor returns coordinates, everything should be okay.

Reply:
@awong-pdx Yeah you're right, this might cause complication. Usually geocode returns an list of object of city names and their coordinates, so their could be multiple places with say name Portland. However, I put the limit query to 1 so that it returns the most populous/common Portland. I think for now, this can stay but if we decide to change, it won't be too difficult to do.

Reply:
Okay, that makes sense!